docs/bug-triage.md

```diff
@@ -0,0 +1,88 @@
+# Bug triage process
+
+## Actions
+
+When triaging a bug in most cases there is a single action required. Actions can be
+executed on an GitHub Issue either by adding the label `action-<action name>` or
+adding `~<action-name>` as the last line of the comment.
+
+For example if an issue is missing information you could comment:
+
+```
+There is not enough information here, and I'm not following the steps-to-reproduce.
+
+~missing-info
+```
+
+You could also for example just add the label `action/priority-regression` to an issue.
+
+## Triaging bugs
+
+### Verify the issue
+
+The first step is to verify if the issue is a valid issue following these questions:
+
+![Bug triaging - Verify!](bug-triage-verify.svg "Bug Triage - Verify")
+
+If an issue is not valid add a comment with some explanation, and add `~<action-name>` as the last line of the comment
+to trigger corresponding action. 
+
+In cases where it is clear that no additional comment is needed you can just add the `action/<action-name>` label. For
+example if the description only states `It doesn't work` then there's not much point in explaining what information is
+missing.
+
+### Prioritize the issue
+
+Second step is to prioritize the bug depending on how common the use-case is, if it's a regression, 
+or not blocking anything, for example a typo:
+
+![Bug triaging - Prioritize!](bug-triage-prioritize.svg "Bug Triage - Prioritize")
+
+When selecting the priority for an issue add the `action-<action-name>` label to the issue.
+
+
+## Missing information
+
+Bugs with insufficient information are assigned the labels `status/missing-info` and `status/auto-expire`.
```

Comment:
Maybe also add "are automatically assigned" if this is done by the bot? For me it is not 100% clear and in the rest of the text the term "automatically" seems to be used mostly to make this explicit.

Reply:
Clarified this section a bit. There's no automatic assignment. However, the bot takes an issue with missing information out of triage, and puts it back into triage if the reporter provides more information.